src/main/kotlin/org/opensearch/commons/alerting/model/AlertContext.kt

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.commons.alerting.model
+
+/**
+ * This model is a wrapper for [Alert] that should only be used to create a more
+ * informative alert object to enrich mustache template notification messages.
+ */
+data class AlertContext(
```

Comment:
What should be the relationship between Alert and AlertContext, a composition or inheritance?

Reply:
In this case, I decided to go with inheritance. An AlertContext is an Alert that contains more information for the purposes of enriching the notification messages. We already return the entire alert document in the notification message ctx; this just allows us to expand upon the information that's already there. If we were not concerned with the size of the alert documents, it would make sense for all of this new info to be within the alert itself.